browser/main/modals/InitModal.js

```diff
@@ -160,6 +147,19 @@ class InitModal extends React.Component {
                 note: note
               })
             })
+          let defaultMarkdownNote = dataApi
```

Comment:
Because it changes order of the first two contents (markdown and snippet).

Before
<img width="315" alt="screen shot 0029-07-07 at 6 37 08 pm" src="https://user-images.githubusercontent.com/8602615/27952463-5305589c-6343-11e7-8a02-d6666885aede.png">


After
<img width="308" alt="screen shot 0029-07-07 at 6 36 59 pm" src="https://user-images.githubusercontent.com/8602615/27952475-598a945c-6343-11e7-8a82-d87fb02c28cb.png">